tests/attestation_plugin_cert/enc/enc.cpp

```diff
@@ -186,46 +189,67 @@ oe_result_t get_tls_cert_signed_with_key(
         public_key_size,
         optional_parameters,
         optional_parameters_size,
-        &output_cert,
-        &output_cert_size);
+        &output_certificate,
+        &output_certificate_size);
     if (result != OE_OK)
     {
-        OE_TRACE_ERROR(" failed with %s\n", oe_result_str(result));
+        OE_TRACE_ERROR(
+            "oe_get_attestation_certificate_with_evidence_v2 failed with %s\n",
+            oe_result_str(result));
         goto done;
     }
 
-    OE_TRACE_INFO("output_cert_size = 0x%x", output_cert_size);
+    OE_TRACE_INFO("output_certificate_size = 0x%x", output_certificate_size);
 
     oe_verifier_initialize();
     // validate cert inside the enclave
-    result = oe_verify_attestation_certificate_with_evidence(
-        output_cert, output_cert_size, enclave_claims_verifier, nullptr);
+    result = oe_verify_attestation_certificate_with_evidence_v2(
+        output_certificate,
+        output_certificate_size,
+        nullptr,
+        0,
+        nullptr,
+        0,
+        &claims,
+        &claims_length);
     OE_TRACE_INFO(
-        "\nFrom inside enclave: verifying the certificate... %s\n",
+        "\nFrom inside enclave: "
+        "oe_get_attestation_certificate_with_evidence_v2 verifying the "
+        "certificate... %s\n",
+        result == OE_OK ? "Success" : "Fail");
+
+    result = enclave_claims_verifier(claims, claims_length, nullptr);
```

Comment:
Similar comment above, consider `goto done` if `result != OE_OK`.
And update comment at Line 29 since `enclave_claims_verifier` is no longer a callback.